src/docs/frontend/index.mdx

```diff
@@ -15,15 +15,15 @@ The frontend codebase is currently located under `src/sentry/static/sentry/app`
 - Name a file meaningfully, based on the how the module's functions, or classes are used or the application section they are used in.
 - Unless necessary, don't use prefixes or suffixes (ie. `dataScrubbingEditModal`, `dataScrubbingAddModal`) instead favor names like `dataScrubbing/editModal`.
 
-### Using `index.(j|t)?(sx)`
+### Using `index.(j|t)sx?$`
```

Comment:
Threw together some quick tests for what this regexp it should/shouldn't match

**Before:**
<img width="671" alt="broken regex" src="https://user-images.githubusercontent.com/187460/160186823-ed385eb9-daf9-414b-b307-0f7b96a00377.png">

**After:**
<img width="660" alt="fixed regex" src="https://user-images.githubusercontent.com/187460/160186820-81c7c0cf-32a6-4ff9-8451-5b51ec4ddc20.png">